heat/core/devices.py

```diff
@@ -132,3 +133,52 @@ def use_device(device=None):
     """
     global __default_device
     __default_device = sanitize_device(device)
+
+
+def _use_envar_device():
+    """Read the environment variable 'HEAT_USE_DEVICE' and return the requested devices.
+    Supported values
+        - cpu: Use CPU only
+        - gpu: Use GPU only
+        - lcpu: GPU global, CPU local
+        - lgpu: CPU global, GPU local
+
+    This function is intended for testing purposes.
+
+    Returns
+    -------
+    torch_device: torch.device
+        The corresponding torch device
+    ht_device : Device, None
+        The local heat device
+    envar: str
+        The value of 'HEAT_USE_DEVICE'
```

Comment:
can you add a quick example to this. just something simple. `HEAT_USE_DEVICE=gpu mpirun.....`